### PR TITLE
"from now" mode for projections

### PR DESCRIPTION
**Pull request by DavidBadura**

### CODE REVIEW

File: src/Projection/Projectionist/DefaultProjectionist.php [hunk 1]
```diff
@@ -593,4 +590,92 @@ private function handleRetryProjections(ProjectionCollection $projections): void
             );
         }
     }
+
+    private function handleFromNowProjections(ProjectionCollection $projections): void
```
Comment (DanielBadura) [inline src/Projection/Projectionist/DefaultProjectionist.php:594]:
```suggestion
    private function activateFromNowProjectionsWithLatestIndex(ProjectionCollection $projections): void
```
Maybe to be more clear?

File: src/Projection/Projectionist/DefaultProjectionist.php [hunk 2]
```diff
@@ -593,4 +590,92 @@ private function handleRetryProjections(ProjectionCollection $projections): void
...
+
+            if (!$projector) {
+                continue;
+            }
+
+            $metadata = $this->metadataFactory->metadata($projector::class);
+
+            if (!$metadata->fromNow) {
+                continue;
+            }
```
Comment (DanielBadura) [inline src/Projection/Projectionist/DefaultProjectionist.php:609]: maybe extract this filtering in a private method to increase readability and reduce the complexity in this method?

**Review by DanielBadura — APPROVED** [review]

File: src/Projection/Projectionist/DefaultProjectionist.php (hunk 1 above)
Comment (DavidBadura) [reply src/Projection/Projectionist/DefaultProjectionist.php:594]: I will do this in a following PR as I have started building the Projectionist differently.

File: src/Projection/Projectionist/DefaultProjectionist.php (hunk 2 above)
Comment (DavidBadura) [reply src/Projection/Projectionist/DefaultProjectionist.php:609]: I will do this in a following PR as I have started building the Projectionist differently.

**Pull request merged**